lib/shelley/0.2-percent-rnd.timelog

```diff
@@ -0,0 +1,235 @@
+1600712573.805894689s -0
+1600712573.926944939s 1000
+1600712574.041849965s 2000
+1600712574.158674401s 3000
+1600712574.238137394s 4000
+1600712574.291298178s 5000
+1600712574.365636401s 6000
+1600712574.503498748s 7000
+1600712574.663584257s 8000
```

Comment:
😅  Perhaps worth adding to `.gitignore`
?
<img width="148" alt="Skärmavbild 2020-09-22 kl  12 13 36" src="https://user-images.githubusercontent.com/304423/93870162-13dd8000-fccd-11ea-85fd-dbf71a9a1440.png">

Reply:
Whoops. Sounds like I a wrong `git add lib` indeed :)